docs/release/release-tasks.md

```diff
@@ -421,6 +421,7 @@ The goal of this task is to keep our tests running in CI stable.
     3. Mark the issue as `release-blocking` if applicable.
 4. Triage periodic GitHub actions failures, with special attention to image scan results;
    Eventually open issues as described above.
+5. Monitor IPv6 testing PR informing jobs (look for `capi-pr-e2e-informing-ipv6-<branch_name>` tab on main and supported releases testgrid dashboards), since they are not part of the any periodic jobs.
```

Comment:
```suggestion
5. Monitor IPv6 testing PR informing jobs (look for `capi-pr-e2e-informing-ipv6-<branch_name>` tab on main and supported releases testgrid dashboards), since they are not part of any periodic jobs.
```